projects/engine-vscode/src/command.ts

```diff
@@ -0,0 +1,35 @@
+import { Plugin, Profile, PluginOptions } from "@remixproject/engine"
+import { Disposable, commands } from "vscode"
+
+export const transformCmd = (name: string, method: string) => `${name}.${method}`
+
+interface CommandOptions extends PluginOptions {
+  transformCmd: (name: string, method: string) => string
+}
+
+export class CommandPlugin extends Plugin {
```

Comment:
Could you give some detail here about the place of `command` in the VScode stack?

Reply:
It's a record of every command you can run in vscode. When you do `ctrl+p` you can look for all the command available.
Omkara needs that to connect command on activation / deactivation of plugins. I'll add a comment "WIP".

Reply:
Thanks for the precision, can you add a class comment too?